src/components/RoundForm.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <div>
+    <h2>Input Rounds</h2>
+    <p>{{ msg }}</p>
+    
+    <input v-model="date" type="text" size="35" placeholder="Enter date, yyyy-mm-dd"><br><br>
```

Comment:
A future todo could be to add a date picker component 🙂

Reply:
Great point!